views/partials/fxa_enabled/fxa_scan.hbs

```diff
@@ -0,0 +1,41 @@
+<main class="scan-results">
+  <div class="section-wrapper breach-container">
+    <!-- scan results page left -->
+    <div class="half scan-results-message">
+      {{> fxa_enabled/scan_results/result_headline}}
+      {{> fxa_enabled/scan_results/result_subhead}}
+      <!-- append "see full report", but only if it's a signed in user scanning from a feautred breach page, with scan results greater than one. -->
+      {{#if featuredBreach}}
+        {{#ifCompare foundBreaches.length ">" 1}}
+          {{#if req.session.user }}
+            {{> fxa_enabled/see_full_report_button}}
+          {{/if}}
+        {{/ifCompare}}
+      {{/if}}
+    </div>
+    <!--scan results page right -->
+    {{> fxa_enabled/scan_results/result_page_right}}
+
+    {{#ifCompare featuredBreach "&&" req.session.user}}
+    <!-- "See Full Report Button" is surfaced above -->
+    {{else}}
+      {{> compromised_accounts }}
+    {{/ifCompare}}
+  </div>
+</main>
+
+{{#ifCompare foundBreaches.length "===" 0}}
+  {{> scan_another_email}}
+  {{> password_tips}}
+{{else}}
+  {{#ifCompare foundBreaches.length ">=" 1}}
+    {{> banners/sign_up_banner class="extra-margin"}}
+  {{/ifCompare}}
+  {{#if fullReport}}
+    {{> what_to_do class="extra-margin"}}
+  {{/if}}
+    {{> what_to_do}}
```

Comment:
This may be suspicious too. I just ran a scan on Heroku (logged in via FxA) and it seems to be duplicating the "What To Do When Your Information Is Exposed in a Data Breach" sections:

![firefox_monitor___scan_results](https://user-images.githubusercontent.com/557895/51948031-e9460180-23db-11e9-9c9d-af0dc12ce7cf.png)